modules/tensor_mechanics/doc/content/documentation/systems/Modules/TensorMechanics/LineElementMaster/LineElementAction.md

```diff
@@ -0,0 +1,69 @@
+# LineElement Action
+
+!syntax description /Modules/TensorMechanics/LineElementMaster/LineElementAction
+
+The LineElement Action is a convenience object that simplifies part of the
+mechanics/dynamics system setup for beam and truss elements.
+
+## Truss elements
+
+Truss element type is chosen by setting `truss = true` in the input block. The following objects can be setup for a truss element by the LineElement Action:
+
+- Sets up displacement variables (if `add_variables` is set to true)
+- Adds StressDivergence Kernels ([StressDivergenceTensorsTruss](/StressDivergenceTensorsTruss.md))
+- Adds Truss material ([LinearElasticTruss](/LinearElasticTruss.md))
```

Comment:
For consistency with the regular TM master action, maybe this shouldn't set up the elasticity and stress calculation material models, because the TM master action doesn't set up their equivalents for solid mechanics problems. The reason why is that those are typically specialized models, and we don't want to have to have all of the inputs for every mechanics model plugged into the action block.  Right now, we only have elastic models available for beams and trusses, but that will hopefully change in the future.

Reply:
That makes sense. I will remove LinearElasticTruss, ComputeElasticityBeam and ComputeBeamResultants from the action.